packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -708,9 +709,10 @@ class _ViewContentState extends State<_ViewContent> {
     return widget.viewBuilder!(suggestions);
   }
 
-  void updateSuggestions() {
+  Future<void> updateSuggestions() async {
+    final Iterable<Widget> suggestions = await widget.suggestionsBuilder(context, _controller);
```

Comment:
Shouldn't you be checking `mounted` here before calling setState?

Otherwise, you'll probably run into `use_build_context_synchronously`.
Also, I'd wrap `updateSuggestions()` above in `unawaited()`, because of `unawaited_futures`.

Reply:
Done.